pkg/kubelet/image_manager_test.go

```diff
@@ -32,14 +32,17 @@ import (
 
 var zero time.Time
 
-func newRealImageManager(t *testing.T, policy ImageGCPolicy) (*realImageManager, *dockertools.FakeDockerClient, *cadvisor.Mock) {
+func newRealImageManager(policy ImageGCPolicy) (*realImageManager, *dockertools.FakeDockerClient, *cadvisor.Mock) {
 	fakeDocker := &dockertools.FakeDockerClient{
 		RemovedImages: util.NewStringSet(),
 	}
 	mockCadvisor := new(cadvisor.Mock)
-	im, err := newImageManager(fakeDocker, mockCadvisor, policy)
```

Comment:
A quick question (which may not be relevant to this PR). Why does we start the background thread so early (in newImageManager)? This is before kubelet was officially created. Can't we start it later, similar to calling StartGarbageCollection()?